applications/serial_lte_modem/src/gnss/slm_at_gnss.c

```diff
@@ -394,29 +396,25 @@ static void fix_rep_wk(struct k_work *work)
 	}
 
 #if defined(CONFIG_SLM_NRF_CLOUD)
-	if (IS_ENABLED(CONFIG_SLM_LOG_LEVEL_DBG)) {
```

Comment:
This change is not aligned with gnss_startup(). Original design is, under CONFIG_SLM_LOG_LEVEL_DBG, SLM enable ALL NMEA masks so all NMEA messages could be logged (which is for customer to debug GPS issue before a fix). While in the other case, if not CONFIG_SLM_LOG_LEVEL_DBG, SLM only enable GGA that is only needed to be signified to nRF Cloud for map drawing. With current change, SLM will send lots of unnecessary NMEA messages to nRF Cloud for location signification.

Reply:
@tomi-font @junqingzou : I must admit that I am not fully following the conversation, but this is keeping me from approving this PR. Have you reached a conclusion whether the current design is sufficient for our use-cases?